src/revlanguage/functions/phylogenetics/ratematrix/Func_wag.cpp

```diff
@@ -72,7 +182,6 @@ Func_wag* Func_wag::clone( void ) const
 
 RevBayesCore::TypedFunction< RevBayesCore::RateGenerator >* Func_wag::createFunction( void ) const
 {
-    
     RevBayesCore::TypedDagNode<RevBayesCore::Simplex>* bf = NULL;
     const RevObject* obj = &this->args[0].getVariable()->getRevObject();
     if (obj == NULL) {
```

Comment:
It seems that obj will never be NULL as the default value of this argument is the empirical frequency vector?

Reply:
This was old code.  I cleaned it up a bit, and it is smaller and simpler now.